client/components/orderable-list/style.scss

```diff
@@ -1,29 +1,32 @@
 .orderable-list {
 	margin: 0;
 
-	&:not( .has-drag-handles ) &__drag-handle {
-		background-image: none;
```

Comment:
Just FYI, we discussed this with @jarekmorawski (p1617803716291900/1617803331.289500-slack-CU6SYV31A) and when the drag handles disappear, the list item contents should stay in place without getting aligned to the left.

Reply:
Thanks for the mention, @waclawjacek. It seems like there's some technical difficulty there. In this case, I think we can leave the drag handle as is even if there's only one item in the list. I think user will quickly realize that there's no point in dragging the element and this way the interface wouldn't break because of the odd space where the icon is supposed to be.

Let me know how you feel about that!

Reply:
To clarify, my changes do not shift the items to the left when the drag handles are not visible
![2021-04-22 08 18 30](https://user-images.githubusercontent.com/273592/115721169-66eee280-a343-11eb-81d4-d2276e48fb81.gif)